repository/src/main/java/org/jbpm/process/workitem/repository/service/RepoService.java

```diff
@@ -59,6 +60,9 @@ public RepoService() {
             services = mapper.readValue(jsonInput,
                                         new TypeReference<List<RepoData>>() {
                                         });
+            // remove all nulls from list if any (can happen due to trailing commans in json)
```

Comment:
Looks good, just small typo: commans -> commas probably. Of course it's up to you whether you change it :)

Reply:
thank you! will update